allocd-port/build/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
wouldn't it be easier to put the following in `allocd-port/.gitignore`:
```
build/*
bin/*
```

Reply:
The intention is to commit an empty directory. I believe git allows me to commit files or directories that contains the files but not an empty directory. I guess this is a common trick to bypass it. I've tried the suggestion. I think it doesn't allow me to add bin or build directories.

Reply:
What's the purpose of those empty directories?
My guess is that some script or tool will use them, would it be possible to have the script/tool create them if they don't already exist?